dev/compose/certified-sync.yaml

```diff
@@ -536,8 +536,10 @@ services:
       - '../nginx/certified-sync/insights.conf:/etc/nginx/nginx.conf:ro'
 
 volumes:
-  var_lib_pulp:
-    name: var_lib_pulp
+  var_lib_pulp_standalone:
+    name: var_lib_pulp_standalone
+  var_lib_pulp_insights:
+    name: var_lib_pulp_insights
```

Comment:
👍🏻 nice! so we dont have conflicts when running multiple profiles in the same system.

we probably need to do that also on `aap` and `community` profiles.

Reply:
Good idea :+1: 
I did this mainly because two profiles shared the same data and that made no sense when testing the sync